internal/version/version.go

```diff
@@ -13,7 +13,7 @@ import (
 // Tag specifies the current release tag. It needs to be manually
 // updated. A test checks that the value of Tag never points to a
 // git tag that is older than HEAD.
-const Tag = "v1.49.0-rc.1"
+const Tag = "v1.49.0-rc.3"
```

Comment:
I missed changing the tag to rc.2 which already exists so this is bumped extra to 3